client/src/main/java/org/apache/rocketmq/client/impl/producer/DefaultMQProducerImpl.java

```diff
@@ -1326,12 +1326,10 @@ public TransactionSendResult sendMessageInTransaction(final Message msg,
                     }
 
                     if (localTransactionState != LocalTransactionState.COMMIT_MESSAGE) {
-                        log.info("executeLocalTransactionBranch return {}", localTransactionState);
-                        log.info(msg.toString());
+                        log.info("executeLocalTransactionBranch return: {} message: {}", localTransactionState, msg);
                     }
                 } catch (Throwable e) {
-                    log.info("executeLocalTransactionBranch exception", e);
-                    log.info(msg.toString());
+                    log.error("executeLocalTransactionBranch exception, message: {}", msg, e);
```

Comment:
It seems that exception stack cannot be printed correctly here,  please check the method  `org.apache.rocketmq.logging.InternalLogger#error(java.lang.String, java.lang.Throwable)`.

Also, `msg` may be very large, logging it in a separate line may be more clear and friendly to GC.

Reply:
Thanks for your review
Logger supports this format, 
maybe we can just print a few key messages instead of the entire message